cli/lsp/tsc.rs

```diff
@@ -1144,6 +1149,50 @@ pub fn request(
   }
 }
 
+type Request = (
+  RequestMethod,
+  ServerStateSnapshot,
+  oneshot::Sender<Result<Value, AnyError>>,
+);
+
+#[derive(Clone)]
+pub struct TSC(mpsc::Sender<Request>);
+
+impl TSC {
```

Comment:
Probably needs to be `Tsc`, like `Url`, as `TSC` made me think this was some sort of generic or constant, which it is neither.